### DISCUSSION TITLE
Sovereign local wallet?

### DISCUSSION THREAD

**Discussion by the42** [filecoin-project/community#84] [Q&A]
Hi, I am interessted in filecoin yet cannot participate with the `lotus daemon` as I do not meet the requirements. I have a question: Will https://github.com/filecoin-project/community/blob/master/projects/wallet/sovereign.md be local software wallet?

Because I do not see much point in having a hardware ledger when all I want to receive and send filecoins to an address. And I do not like the idea (read: trust) third (web) parties to interact with filecoins I own.

If https://github.com/filecoin-project/community/blob/master/projects/wallet/sovereign.md is going to be a local wallet please consider to make it cross-platform.
If I do not understand the purpose of sovereign ... are there any local wallet options to receive and send filecoins besides lotus?

**Comment by jimmylee** [comment]
Which platform would you like Sovereign to support?

**Reply by the42** [reply]
I for one would vote for Linux - at least that's the platform I am using and  I guess it is the easier target.

**Reply by jimmylee** [reply]
Okay I can cut a release, just know its experimental technology :-)

**Reply by the42** [reply]
Too cool, looking forward!

**Comment by whyrusleeping** [comment]
we are holding off on adding support for local non-ledger keys until we can get a security audit on the code. I dont want to be responsible for anyone losing funds because a rogue javascript package managed to steal your keyfiles.

When we get things to that point, we will follow up. Until then, you can use sovereign with a ledger
